InternetHospital.WebApi/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "User ID = postgres;Password=12345678;Server=localhost;Port=5432;Database=HospitalDB;Integrated Security=true; Pooling=true;"
+    "DefaultConnection": "User ID = postgres;Password=1111;Server=localhost;Port=5432;Database=HospitalDB;Integrated Security=true; Pooling=true;"
```

Comment:
Remove this file from pull request

Reply:
I will delete it while resolving conflicts